src/backend/distributed/executor/multi_router_executor.c

```diff
@@ -330,10 +481,11 @@ ExecuteTaskAndStoreResults(QueryDesc *queryDesc, Task *task,
 		PlannedStmt *planStatement = queryDesc->plannedstmt;
 		MultiPlan *multiPlan = GetMultiPlan(planStatement);
 		Query *query = multiPlan->workerJob->jobQuery;
+		Oid relid = ((RangeTblEntry *) linitial(query->rtable))->relid;
 		StringInfo queryStringInfo = makeStringInfo();
 
 		ExecuteMasterEvaluableFunctions(query);
-		DeparseShardQuery(query, task, queryStringInfo);
+		deparse_shard_query(query, relid, task->anchorShardId, queryStringInfo);
```

Comment:
It was two lines to inline and was only used in exactly this one place; when reordering I didn't see a natural place for that helper function to live in this file, and it lacked a function comment.

I figured if it was simple enough to omit a comment and used exactly in one place I'd just inline it. Seemed weird to have to jump down several functions in the file to just see this little helper that just wrapped a call to `deparse_shard_query`.